templates/docker-compose.yml.j2

```diff
@@ -63,6 +63,7 @@ services:
 {% if nim_waku_rln_relay_dynamic %}
       --rln-relay-eth-account-address={{ nim_waku_rln_relay_eth_account_address }}
       --rln-relay-eth-contract-address={{ nim_waku_rln_relay_eth_contract_address }}
+      --rln-relay-tree-path={{ nim_waku_rln_relay_tree_path }}
```

Comment:
Please move this below or above the `--rln-releay-eth-...` lines.

Reply:
Addressed in [122c459](https://github.com/status-im/infra-role-nim-waku/pull/11/commits/122c45979c152c538adf188bca1b137c837f620b), moved it above the dynamic conditional since it does not depend on the membership group type